From 5f7d217e7372bb4085e76332606dd33478b342cf Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 1 Sep 2014 20:15:33 -0400
Subject: [powerpc] perf/hv-24x7: Remove [static 4096], sparse chokes on it

Message-id: <1409602534-23951-15-git-send-email-gduarte@redhat.com>
Patchwork-id: 90128
O-Subject: [RHEL7.1 PATCH BZ 947159 14/15] powerpc/perf/hv-24x7: Remove [static 4096], sparse chokes on it
Bugzilla: 947159
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=947159
Upstream Status: 1ee9fcc1a021e54454a23b107a47d10fb603508c

commit 1ee9fcc1a021e54454a23b107a47d10fb603508c
Author: Cody P Schafer <cody@linux.vnet.ibm.com>
Date:   Tue Apr 15 10:10:54 2014 -0700

    powerpc/perf/hv-24x7: Remove [static 4096], sparse chokes on it

    Signed-off-by: Cody P Schafer <cody@linux.vnet.ibm.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/perf/hv-24x7.c b/arch/powerpc/perf/hv-24x7.c
index 3e8f60a..95a67f8 100644
--- a/arch/powerpc/perf/hv-24x7.c
+++ b/arch/powerpc/perf/hv-24x7.c
@@ -170,7 +170,7 @@ static unsigned long h_get_24x7_catalog_page_(unsigned long phys_4096,
    index);
 }
 
-static unsigned long h_get_24x7_catalog_page(char page[static 4096],
+static unsigned long h_get_24x7_catalog_page(char page[],
           u32 version, u32 index)
 {
  return h_get_24x7_catalog_page_(virt_to_phys(page),
-- 
1.7.1